libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -819,6 +819,18 @@ void SRV_Channels::set_slew_rate(SRV_Channel::Function function, float slew_rate
     _slew = new_slew;
 }
 
+// update channels last_scaled_output to match value
+void SRV_Channels::set_slew_last_scaled_output(SRV_Channel::Function function, float value)
```

Comment:
Maybe "reset_slew_limit", that is a little more inline with what we would call it for the PIDs. It would set the current and last output in one call.